lib/declare_schema/model/index_definition.rb

```diff
@@ -64,6 +64,8 @@ def for_table(table_name, ignore_indexes, connection)
         end
 
         def default_index_name(table_name, columns)
+          return long_index_name(table_name, columns) if DeclareSchema.current_adapter == 'postgresql'
+
```

Comment:
Rather than early `return`, can you make the rest of this method a new private method, say `longest_valid_index_name`? Then this method will become
```ruby
def default_index_name(table_name, columns)
  if DeclareSchema.current_adapter == 'postgresql'
    long_index_name(table_name, columns)
  else
    longest_valid_index_name(table_name, columns)
  end
end
```

Reply:
Get idea, done 057d3c26a57ca6a24f6a54c6d4e97b31f91330c3